airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -2235,6 +2240,12 @@ def _emit_running_ti_metrics(self, session: Session = NEW_SESSION) -> None:
 
         self.previous_ti_running_metrics = ti_running_metrics
 
+    @provide_session
+    def _emit_running_dags_metric(self, session: Session = NEW_SESSION) -> None:
+        stmt = select(func.count()).select_from(DagRun).where(DagRun.state == DagRunState.RUNNING)
+        running_dags = session.scalar(stmt)
+        Stats.gauge("executor.running_dags", running_dags)
```

Comment:
Sorry, I'm late to this PR. Why are we emitting this metric under `executor` when executors have nothing to do with dag level scheduling? Shouldn't this be under `scheduler`?  The config you're using above even says `scheduler.dagrun_metrics_interval`